DOCUMENTATION.md

```diff
@@ -0,0 +1,232 @@
+# Amazon Connect Chat Interface Documentation
+
+Looking to build a custom chat-interface for Amazon Connect Chat? Learn how to set up this package for local development, see overview of React components, and enable/configure existing features.
+
+> For additional common features (e.g. customer name from a form), check out the [customChatWidget](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/customChatWidget) example.
+
+## Table of contents
+
+- [Documentation](#documentation) - additional documentation reference
+- [Features](#features) - overview of global configuration options (logger, feature flags, etc)
+- [Customization](#customization)- high level overview of some of the major components
+- [Components](#customization) - details on how to enable/use various features
+- [Usage](#usage) - instructions to use `amazon-connect-chat-interface.js` file in an html page
+
+## Official AWS Documentation
+
+- [StartChatConact API Backend (CFN Template)](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/startChatContactAPI)
+- [Custom Chat Widget Example App](https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/customChatWidget)
+- [Amazon Connect Chat Experience Admin Guide](https://docs.aws.amazon.com/connect/latest/adminguide/enable-chat-in-app.html)
+- [Amazon Connect Open Source Documentation](https://docs.aws.amazon.com/connect/latest/adminguide/download-chat-example.html)
+- [Out-of-box Amazon Connect Chat Widget Setup](https://docs.aws.amazon.com/connect/latest/adminguide/add-chat-to-website.html)
+
+## Features
```

Comment:
All these links need to be updated

Reply:
This links will work, but I had to split into separate CR: https://github.com/amazon-connect/amazon-connect-chat-interface/pull/150

Reply:
Any reason for splitting up the PRs?  Ideally we should merge all dependent changes together if possible to avoid a broken state (even for documentation).